src/pages/SearchPage.js

```diff
@@ -34,208 +37,194 @@ const propTypes = {
-                            />
-                        </View>
-                    </>
-                )}
-            </ScreenWrapper>
-        );
-    }
+                <Text>
+                    {translate(`referralProgram.${CONST.REFERRAL_PROGRAM.CONTENT_TYPES.REFER_FRIEND}.buttonText1`)}
+                    <Text
+                        color={theme.success}
+                        style={themeStyles.textStrong}
+                    >
+                        {translate(`referralProgram.${CONST.REFERRAL_PROGRAM.CONTENT_TYPES.REFER_FRIEND}.buttonText2`)}
+                    </Text>
+                </Text>
+                <Icon
+                    src={Info}
+                    height={20}
+                    width={20}
+                />
+            </PressableWithoutFeedback>
+        </View>
+    );
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom={false}
+            testID={SearchPage.displayName}
+            onEntryTransitionEnd={handleScreenTransitionEnd}
+        >
+            {({didScreenTransitionEnd, safeAreaPaddingBottomStyle}) => (
+                <>
+                    <HeaderWithBackButton title={translate('common.search')} />
+                    <View style={[themeStyles.flex1, themeStyles.w100, safeAreaPaddingBottomStyle]}>
+                        <SelectionList
+                            sections={didScreenTransitionEnd && isOptionsDataReady ? sections : []}
+                            textInputValue={searchValue}
+                            textInputLabel={translate('optionsSelector.nameEmailOrPhoneNumber')}
+                            textInputHint={offlineMessage}
+                            onChangeText={onChangeText}
+                            headerMessage={headerMessage}
+                            onLayout={searchRendered}
+                            autoFocus
+                            onSelectRow={selectReport}
+                            showLoadingPlaceholder={!didScreenTransitionEnd || !isOptionsDataReady}
+                            footerContent={footerRender}
+                        />
+                    </View>
+                </>
+            )}
+        </ScreenWrapper>
+    );
 }
 
 SearchPage.propTypes = propTypes;
 SearchPage.defaultProps = defaultProps;
 SearchPage.displayName = 'SearchPage';
 
-export default compose(
-    withLocalize,
-    withWindowDimensions,
-    withNetwork(),
-    withOnyx({
-        reports: {
-            key: ONYXKEYS.COLLECTION.REPORT,
-        },
-        personalDetails: {
-            key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-        },
-        betas: {
-            key: ONYXKEYS.BETAS,
-        },
-        isSearchingForReports: {
-            key: ONYXKEYS.IS_SEARCHING_FOR_REPORTS,
-            initWithStoredValues: false,
-        },
-    }),
-    withThemeStyles,
-)(SearchPage);
+export default withOnyx({
+    reports: {
+        key: ONYXKEYS.COLLECTION.REPORT,
+    },
+    personalDetails: {
```

Comment:
can we use `usePersonalDetails()` instead of this key?

Reply:
oh, I didn't even know this hook exists :)

Reply:
oh it does and it yells `performanceee` 😂